src/ng/directive/input.js

```diff
@@ -1530,6 +1530,11 @@ function rangeInputType(scope, element, attr, ctrl, $sniffer, $browser) {
 
     // TODO(matsko): implement validateLater to reduce number of validations
     if (minAttrType === 'min') {
+      var elVal = element.val();
+      // IE11 doesn't set the el val correctly if the maxVal is less than the element value
```

Comment:
This comment doesn’t match the surrounding code. Bad copy-paste from [line 1572](https://github.com/angular/angular.js/blob/0dbe342417f8e0357f72314e2375f62ddaaec6ab/src/ng/directive/input.js#L1572)?